zzz_generated/controller-clusters/zap-generated/CHIPClusters.cpp

```diff
@@ -162,8 +164,11 @@ CHIP_ERROR AdministratorCommissioningCluster::OpenBasicCommissioningWindow(Callb
                                          AdministratorCommissioning::Commands::Ids::OpenBasicCommissioningWindow,
                                          (app::CommandPathFlags::kEndpointIdValid) };
 
-    SuccessOrExit(err = app::InteractionModelEngine::GetInstance()->NewCommandSender(&sender));
+    CommandSenderHandler sender(Platform::New<app::CommandSender>(mDevice->GetInteractionModelDelegate()));
```

Comment:
I'm still not certain why we need to use `Platform::New` and a custom deleter. Why not just `make_unique`?

Reply:
To make sure we use the right heap?